packages/astro/src/core/errors/errors-data.ts

```diff
@@ -393,6 +393,22 @@ See https://docs.astro.build/en/guides/server-side-rendering/ for more informati
 			`Could not render \`${componentName}\`. No matching import has been found for \`${componentName}\`.`,
 		hint: 'Please make sure the component is properly imported.',
 	},
+	/**
+	 * @docs
+	 * @description
+	 * A `prerender` export was detected, but the value was not statically analyzable. Values computed at runtime are not supported, so `export const prerender` only supports the exact values `true` or `false`.
```

Comment:
Not a huge change here but small wording tweak:

```suggestion
	 * A `prerender` export was detected, but the value was not statically analyzable. Values computed at runtime are not supported, so `export const prerender` can only be set to `true` or `false`. Variables are not supported.
```